### PR TITLE
feat: style mobile listing cards

### PR DESCRIPTION
**Pull request by ifranch**
# Pull Request Template

## Description

- Change location of "See details" button for desktop vs mobile
- Update margins and padding for mobile listing card as well as desktop where needed
- Round the corners of the image card
- Update font sizes and colors according to Figma specs
- Add border to the bottom of each listing card
- Center listing card content to the left for mobile

Mobile before:
image

Mobile after:
image

Desktop before:
image

Desktop after:
image

iPad before:
image

iPad after:
image

## How Can This Be Tested/Reviewed?

Go to the listings page and try using different devices with the chrome developer tools device emulator

### CODE REVIEW

**Review by phattya — APPROVED** [review]
@ifranch awesome!! It's looking so good! Great job!!!

I did run into this error when on mobile in map only view, when clicking on a pin. Nothing happened...so no card pop-up and the listings drawer didn't move up either. This might be a different PR and unrelated to this.
image

File: doorway-ui-components/src/blocks/ImageCard.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
 .image-card {
   /* Component Variables */
   --default-background-color: var(--bloom-color-gray-500);
-  --border-radius: 0px;
+  --border-radius: 6px;
```
Comment (lisunshiny) [inline doorway-ui-components/src/blocks/ImageCard.scss:7]: `--bloom-rounded-md` or `lg` perhaps? (neither is exactly 6px but both are close --whichever looks better)

File: doorway-ui-components/src/page_components/listing/ListingCard.scss [hunk 2]
```diff
@@ -14,11 +15,17 @@
   max-width: var(--max-width);
   margin: var(--margin);
   position: relative;
+  border-bottom: 1px solid var(--bloom-color-gray-450);
```
Comment (lisunshiny) [inline doorway-ui-components/src/page_components/listing/ListingCard.scss:18]: supernit lol this is `--bloom-border-1`

File: doorway-ui-components/src/page_components/listing/ListingCard.tsx [hunk 3]
```diff
@@ -89,6 +91,27 @@ const ListingCard = (props: ListingCardProps) => {
     }
   }
 
+  const [isDesktop, setIsDesktop] = useState(true)
+
```
Comment (lisunshiny) [inline doorway-ui-components/src/page_components/listing/ListingCard.tsx:95]: oo this is copypasta from another part of the codebase -- sg

**Review by lisunshiny — APPROVED** [review]
tyvm!

**Comment by ifranch** [comment]
> @ifranch awesome!! It's looking so good! Great job!!!
> 
> I did run into this error when on mobile in map only view, when clicking on a pin. Nothing happened...so no card pop-up and the listings drawer didn't move up either. This might be a different PR and unrelated to this. image

Yep the fix for that is coming in a different PR. This PR didn't introduce it, it's from https://github.com/metrotranscom/doorway/pull/149

**Pull request merged**